resources/default_pipeline_environment.yml

```diff
@@ -327,8 +327,14 @@ steps:
     logSuccessfulMavenTransfers: false
   mtaBuild:
     buildTarget: 'NEO'
+    buildTool: classic
+    platform: 'cf'
```

Comment:
do we need separate options for build target and platform? should they have different defaults?

Reply:
I added `platform` as this is the option name in the new tool. Also the new tool has `CF` as default while we use `NEO` for the old tool.
This was we can do a smooth transition and respecting the *new* default and not breaking anything. 
To my understanding `buildTool: classic` and `buildTarget` will go away some time..